mycore-base/src/main/java/org/mycore/common/xml/MCREntityResolver.java

```diff
@@ -274,4 +279,13 @@ public InputSource newInputSource() {
             return is;
         }
     }
+
+    private static class CatalogEntityIdentifier {
+        private String systemId, publicId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4630175)